Explorer/Assets/Scenes/Skybox/SimpleCamera.cs

```diff
@@ -0,0 +1,218 @@
+﻿//===========================================================================//
+//                       FreeFlyCamera (Version 1.2)                         //
+//                        (c) 2019 Sergey Stafeyev                           //
+//===========================================================================//
+
+using UnityEngine;
+
+[RequireComponent(typeof(Camera))]
+public class SimpleCamera : MonoBehaviour
```

Comment:
Its not needed, it was added as part of the Test scene for the artist to move the camera around. It can be replaced by the free fly camera of cinemachine.

Reply:
This is a simple camera used for the Skybox test scene, so content and art team can move freely in during playmode without having to instantiate the CharacterCamera or the player